spec/features/votes/visitor_cannot_vote_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'visitor views review show' do
+  context 'as a guest' do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:restaurant) { FactoryGirl.build(:restaurant, user: user) }
```

Comment:
You can just create a restaurant without building a specific user in this case. You aren't logging in, for example.